finance.md

```diff
@@ -35,3 +45,13 @@ Learn competent trading strategies from a veteran of crypto twitter. He also pro
 Website - https://www.youtube.com/channel/UCH13wWEgjCwITVnWvtqB5cA/videos
 
 [![trading room's youtube channel](https://user-images.githubusercontent.com/4047597/66118879-13983080-e5f5-11e9-8491-8148903f9e66.png)](https://www.youtube.com/channel/UCH13wWEgjCwITVnWvtqB5cA/videos)
+
+
+### Professional Stock Trading Course by Adam Khoo
+
+This is a beginner friendly Stock Trading Course by Adam Khoo designed to reveal all the tools and strategies to trade stocks confidently and profitably.
+
+Website - https://www.youtube.com/watch?v=slBxM4J3BEA
```

Comment:
Thanks for this recommendation. But it looks like this is a single video in the course. Is there a playlist for all the lessons? I found this but I am not sure if this is the best resource to link to.  https://www.youtube.com/watch?v=BBLJsZst9-E&list=PLl_EHtw2h7QSPCKIDGme-kBgoCiFxUQ5Q

If you can make a more compact playlist covering all the videos that cover Stock Trading, in the correct sequence, it would be welcome.

Reply:
If we keep entire playlist there could be too many videos a user need to go through. 
Instead, i picked a single one which covers the essence in 1.5 hours.

Reply:
Thanks @prashanttejura. I have merged it.